ui/emails/mjml/user-invite.mjml

```diff
@@ -40,7 +40,7 @@
           <mj-text font-weight="400" padding-bottom="8px">Verification code</mj-text>
         </mj-column>
         <mj-column width="70%">
-          <mj-text font-size="24px" padding-bottom="8px" font-weight="275" line-height="32px">{{slice .RecoveryCode 0 3}} {{slice .RecoveryCode 3 6}}</mj-text>
+          <mj-text font-size="24px" padding-bottom="8px" font-weight="275" line-height="32px">{{ if .RecoveryCode }}{{ if eq (len .RecoveryCode) 6 }}{{slice .RecoveryCode 0 3}} {{slice .RecoveryCode 3 6}}{{ else }}{{ .RecoveryCode }}{{ end }}{{ end }}</mj-text>
```

Comment:
Ah right, the design calls for a space in the middle e.g. `123 456` but if we don't know that it will be a six digit code maybe we shouldn't try and format it.